packages/tabs/src/useTabs.ts

```diff
@@ -9,16 +9,46 @@ import { useState } from 'react';
 import { useSelection } from '@zendeskgarden/container-selection';
 import { generateId } from '@zendeskgarden/container-utilities';
 
-function requiredArguments(arg, argStr, methodName) {
+interface IUseTabsPropGetters {
+  getTabProps: <T>(options?: T & IGetTabPropsOptions) => T & IGetTabPropsOptions;
+  getTabPanelProps: <T>(options?: T & IGetTabPanelProps) => IGetTabPanelProps;
+  getTabListProps: <T>(options: T) => T & React.HTMLProps<any>;
+}
+
+interface IGetTabPropsOptions extends React.HTMLProps<any> {
+  index?: number;
+}
+
+interface IGetTabPanelProps extends React.HTMLProps<any> {
+  index?: number;
+  item?: any;
+}
+
+export interface IUseTabsReturnValue extends IUseTabsPropGetters {
```

Comment:
This should extend the return state values provided by `useSelection`

Reply:
Oh yes that makes sense. I did not have to redefine Updated, thank you!